admin/class-perfecty-push-admin.php

```diff
@@ -496,6 +496,29 @@ public function show_admin_notice() {
 
 			delete_transient( 'perfecty_push_admin_notice' );
 		}
+
+		$cron_check_status = Perfecty_Push_Cron_Check::get_cron_status();
+
+		if ( ! $cron_check_status ) {
+
+			printf(
+				'<div class="notice notice-warning is-dismissible"><b>' .
```

Comment:
You can use this `Class_Perfecty_Push_Lib_Utils::show_message()`, no need of html, like:

https://github.com/perfectyorg/perfecty-push-wp/blob/afa86988294c8abcafcc1704f06639d31ec78a0b/perfecty-push.php#L80